### DISCUSSION TITLE
Using Keyviz with OBS

### DISCUSSION THREAD

**Discussion by UUoocl** [mulaRahul/keyviz#62] [Show and tell]
Thank you for this awesome tool! I tested Keyviz today, and found it incredibly useful and intuitive.
2 Tips I think Users can benefit from.
1. Keyviz should be added to an OBS scene as a "Game Capture Source", for the transparency effect to work.  
2. Use the Keyviz settings menu to have the keys appear in the scene, but not on the screen.

I made a video to show how to add Keyviz to an OBS scene.
 [Add Keyviz to an OBS scene.](https://rebrand.ly/14a5fut)

Thanks again!

**Comment by mulaRahul** [comment]
> I made a video to show how to add Keyviz to an OBS scene.
[Add Keyviz to an OBS scene.](https://rebrand.ly/14a5fut)

Cool! Thanks for sharing the tutorial! Most people will use keyviz alongside OBS and adding it as an _OBS Source_ unlocks more customization. 

In the upcoming releases, I'm planning to add the "**Follow Mouse**" feature as suggested in #54, and #47. Then it'll not be a problem even if the screen is zoomed.

**Reply by careerprincemichaelfandino-glitch** [reply]
Hey @mulaRahul, thank you so much for this useful tool that you created. I'm really looking forward to a feature in keyviz where I can decide whether to show the visuals in my actual desktop or only in OBS studio.
Sometimes, if the key visuals are in my actual screen, it covers some things in my screen. I want the keystroke appearance to be shown in OBS only and not in my actual desktop. I hope this make sense. Thank you so much Rahul!
